src/llm_utils.py

```diff
@@ -5,178 +5,136 @@
 from google import genai
 from typing import List, Dict
 
+from github_utils import get_valid_hunks
+
 def analyze_code_changes(diff_content: str) -> List[Dict]:
     """
     Analyze code changes using OpenAI's GPT model
```

Comment:
Comment explains the function's purpose.
```suggestion
|
```
N/A